amr-wind/overset/TiogaInterface.H

```diff
@@ -93,12 +94,15 @@ public:
 
     AMROversetInfo& amr_overset_info() { return *m_amr_data; }
 
-    ScratchField& qvars_cell() { return *m_qcell; }
-    ScratchField& qvars_node() { return *m_qnode; }
+    ScratchField& qvars_cell() { return *m_qcell_host; }
```

Comment:
do you want to add a `_host` to the function names for these 2 functions so it is clear it returns a host object?

Reply:
I left the function name as it was since it is called from AMRTiogaIface from inside Exawind driver, and I wanted to avoid making a change in Exawind source. I can go ahead and make that change.